default/python/AI/freeorion_tools/charts_handler.py

```diff
@@ -15,14 +15,14 @@ def charting_text():
     planet = None
     if planet_id is not None:
         planet = universe.getPlanet(planet_id)
-    aggression_name = fo.aggression.values[foAI.foAIstate.aggression].name
+    aggression_name_local = aggression_name[foAI.foAIstate.character]
```

Comment:
> Perhaps we should do a day in each devs mother tongue to put the English in their place.

Brilliant idea - @Cjkjvfnby surely is going to have a blast with us trying to communicate in Russian... 😂